FilmFestivalLoader/Imagine/parse_imagine_html.py

```diff
@@ -345,67 +358,76 @@ def add_paragraph(self):
 
     def set_article(self):
         descr_threshold = 512
-        article = '\n\n'.join(self.article_paragraphs)
-        if self.alt_description is None:
-            self.article = article
-        elif len(self.article_paragraphs) > 0:
+        if self.article_paragraphs:
             self.description = self.article_paragraphs[0]
             if len(self.description) > descr_threshold:
                 self.description = self.description[:descr_threshold] + '…'
-            self.article = f'{self.alt_description}\n\n{article}'
+            self.article = '\n\n'.join(self.article_paragraphs)
         else:
-            self.description = self.film.title
+            self.description = ''
             self.article = ''
         if len(self.film_property_by_label):
             properties = [f'{key}: {value}' for (key, value) in self.film_property_by_label.items()]
             metadata = '\n'.join(properties)
             self.article += f'\n\n{metadata}'
 
     def add_film_info(self):
-        if len(self.description) == 0:
-            self.description = self.film.title
-        film_info = FilmInfo(self.film.film_id, self.description, self.article)
-        self.festival_data.filminfos.append(film_info)
+        if self.description and self.article:
+            film_info = FilmInfo(self.film.film_id, self.description, self.article,
+                                 metadata=self.film_property_by_label)
+            self.festival_data.filminfos.append(film_info)
```

Comment:
Please leave out the `and self.article` case.
The description is read from the `film_info` files, but the article isn't used yet in Django.